src/lib_ccaligner/read_wav_file.cpp

```diff
@@ -105,6 +105,7 @@ bool WaveFileData::decode()     //decodes the wave file
     if(format != "WAVE")
     {
         FATAL(EXIT_INVALID_FILE, "Invalid WAV file format : %s ", format.c_str());
+        LOG("File format couldn't be identified as WAV");
```

Comment:
After FATAL, the function won't continue further and hence this LOG will never be executed :)

Reply:
Oh, damn! I should have noticed this. I'll do the changes!